fence/agents/gce/fence_gce.py

```diff
@@ -99,7 +100,7 @@ def main():
 	run_delay(options)
 
 	try:
-		credentials = oauth2client.GoogleCredentials.get_application_default()
+		credentials, _ = google.auth.default()
```

Comment:
This seems to not be available in a lot of distros yet, so I think we should move the import-line from line 9 to this try-statement, and have a fallback to use the old library (except ImportError:).

Reply:
Ack, I just noticed that googleapiclient already do that if I use the default credentials [1]

I am fixing this PR, sorry for the noise.

[1] https://github.com/google/google-api-python-client/blob/master/googleapiclient/_auth.py#L39